website/docs/docs/deploy/model-notifications.md

```diff
@@ -42,22 +44,26 @@ groups:
   - name: finance
     description: "Models related to the finance department"
     owner:
-      # 'name' or 'email' is required
+      # Email is required to receive model-level notifications, additional properties are also allowed.
       name: "Finance Team"
       email: finance@dbtlabs.com
-      slack: finance-data
+      favorite_food: donuts
 
   - name: marketing
     description: "Models related to the marketing department"
     owner:
       name: "Marketing Team"
       email: marketing@dbtlabs.com
-      slack: marketing-data
+      favorite_food: jaffles
```

Comment:
This is for clarity and is optional, but reduces the likelihood people misbelieve that Slack notifs are supported

Reply:
good callout! i assumed slack was supported when reading